src/Agent.Worker/Release/WellKnownReleaseVariables.cs

```diff
@@ -0,0 +1,26 @@
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
```

Comment:
yeah, I moved most of the release specific constants to that constant file.. The left over here is confusing.. In the old code probably we did not have access to these variables probably thats why we just redefined. I have already added a TODO to clean this up. I will clean them up later.